changelog/bug-1603391.md

```diff
@@ -0,0 +1,3 @@
+level: silent
+reference: bug 1603391
```

Comment:
This certainly warrants mention as an API change.  If that's a breaking change, then `major`.